app/car.py

```diff
@@ -0,0 +1,14 @@
+import math
+
+
+class Car:
+    def __init__(self, brand: str, fuel: float) -> None:
+        self.brand = brand
+        self.fuel = fuel
+
+    def calculate_road_cost(self, customer_loc: list,
+                            shop_loc: list, fuel_price: float) -> float:
```

Comment:
The parameters 'customer_loc' and 'shop_loc' should have more descriptive names indicating that they are coordinates, such as 'customer_coordinates' and 'shop_coordinates'. This aligns with the checklist item: 'Use descriptive and correct variable names.'